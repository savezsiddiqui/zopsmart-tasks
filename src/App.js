import React from 'react'
import Home from './components/task2/Home/Home'
import Board from './components/task2/Board/Board'
import Todo from './components/task1/Todo/Todo'
import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom'
import './App.css'

const App = () => {
  return (
    <BrowserRouter>
      <div className="content">
        <Switch>
          <Route exact path="/">
            <Redirect to="/boards" />
          </Route>
          <Route exact path="/Todo">
            <Todo />
          </Route>
          <Route exact path="/boards">
            <Home />
          </Route>
          <Route exact path="/board/:boardId">
            <Board />
          </Route>
        </Switch>
      </div>
    </BrowserRouter>
  )
}

export default App
