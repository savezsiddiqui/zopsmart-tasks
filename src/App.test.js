import React from 'react'
import renderer from 'react-test-renderer'
import App from './App'

test('renders app', () => {
  const tree = renderer.create(<App />).toJSON()
  expect(tree).toMatchSnapshot()
})
