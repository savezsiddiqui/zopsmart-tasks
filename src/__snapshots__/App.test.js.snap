// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app 1`] = `
<div
  className="content"
>
  <div
    className="root"
  >
    <div
      className="header"
    >
      <h2
        className="text"
      >
        Boards
      </h2>
      <div
        className="margin-start"
      >
        <input
          className="custom-input"
          onChange={[Function]}
          placeholder="filter boards"
        />
      </div>
    </div>
    <div
      className="board-container"
    >
      <div
        className="board"
      >
        <p
          className="text"
          onClick={[Function]}
        >
          + add board
        </p>
      </div>
    </div>
  </div>
</div>
`;
